repos/os/include/net/ipv4.h

```diff
@@ -66,8 +66,8 @@ class Net::Ipv4_packet
 		 ** IPv4 header fields **
 		 ************************/
 
-		unsigned         _version         : 4;
 		unsigned         _header_length   : 4;
+		unsigned         _version         : 4;
```

Comment:
I think this was a byte-order issue. Apparently, bitfields are affected by the endianess so both fields where shuffled in the memory layout. I tested this with wireshark.